integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -43,10 +55,463 @@ class AuthorizationServerIT extends ITSupport {
+        assertThat(updatedAuthorizationServer.getId(), equalTo(createdAuthorizationServer.getId()))
+        assertThat(updatedAuthorizationServer.getDescription(), equalTo("Updated Test Authorization Server"))
+    }
+
+    @Test
+    void deleteAuthorizationServerTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+        assertThat(client.getAuthorizationServer(createdAuthorizationServer.getId()), notNullValue())
+
+        createdAuthorizationServer.deactivate()
+        createdAuthorizationServer.delete()
+
+        sleep(2000)
+
+        try {
+            client.getAuthorizationServer(createdAuthorizationServer.getId())
+        }
+        catch (ResourceException e) {
+            assertThat(e.status, equalTo(404))
+        }
+    }
+
+    // Policy operations
+
+    @Test
+    void listAuthorizationServerPoliciesTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        Policy policy = client.instantiate(Policy)
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition).setInclude(["ALL_CLIENTS"])))
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        Policy createdPolicy = createdAuthorizationServer.createPolicy(policy)
+        registerForCleanup(createdPolicy)
+
+        assertPresent(createdAuthorizationServer.listPolicies(), createdPolicy)
+    }
+
+    @Test
+    void getAuthorizationServerPolicyTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        Policy policy = client.instantiate(Policy)
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition).setInclude(["ALL_CLIENTS"])))
```

Comment:
_should_ this be an enum?

Reply:
See my comment below. Per your other comment, I would consider moving `ALL_CLIENTS` enum out of `MetadataPublishEnum` so it can be used here. 

I will note this down do this as part of a list of improvements to enum and constants as part of a separate PR.

Reply:
https://oktainc.atlassian.net/browse/OKTA-305898 will address this.